cli/src/pcluster/cli/commands/configuration_converter.py

```diff
@@ -0,0 +1,608 @@
+#  Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+#  with the License. A copy of the License is located at http://aws.amazon.com/apache2.0/
+#  or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+#  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+#  limitations under the License.
+import errno
+import json
+import logging
+import os
+import re
+import stat
+import sys
+from collections import OrderedDict
+
+import argparse
+import configparser
+import yaml
+
+from pcluster.utils import get_partition
+
+
+class ConfigDumper(yaml.Dumper):
+    def increase_indent(self, flow=False, indentless=False):
+        return super(ConfigDumper, self).increase_indent(flow, False)
+
+
+def _role_name_to_arn(role_name):
+    return "arn:{0}:iam::aws:role/{1}".format(get_partition(), role_name)
```

Comment:
is this correct?or aws needs to be replaced by the account id?

Reply:
You are right, aws needs to be replaced by the account id. Fixed